config/envs/default.py

```diff
@@ -99,6 +99,7 @@ class DefaultConfig(object):
     FSD_SESSION_TIMEOUT_SECONDS = CommonConfig.FSD_SESSION_TIMEOUT_SECONDS
     WTF_CSRF_TIME_LIMIT = CommonConfig.WTF_CSRF_TIME_LIMIT
     CREATE_APPLICATION_ON_ACCOUNT_CREATION = False
+    SESSION_ROLES = True
```

Comment:
Can we call this something more descriptive to the actual use case e.g.
```
ALLOW_ASSESSMENT_LOGIN_VIA_MAGIC_LINK
```
or similar?